test/lit/passes/outlining.wast

```diff
@@ -1109,3 +1109,56 @@
     unreachable
   )
 )
+
```

Comment:
Strangely, running update_lit_checks.py on this test file removes the asserts from the below module. I produced these asserts from putting this module into a separate, new file.

Reply:
It turns out that having spaces before `(module` causes the one of the scripts to not recognize that there is a module here. Delete those spaces and the checks reappear. (There are some other indentation oddities to fix as well.)

Reply:
done, thanks